service/storage_service.cc

```diff
@@ -4409,6 +4409,11 @@ storage_service::describe_ring(const sstring& keyspace, bool include_only_local_
     return locator::describe_ring(_db.local(), _gossiper, keyspace, include_only_local_dc);
 }
 
+future<std::vector<storage_service::token_range_endpoints>>
+storage_service::describe_ring_for_table(std::string_view ks_name, std::string_view cf_name) const {
+    return locator::describe_ring_for_table(_db.local(), _gossiper, ks_name, cf_name);
+}
```

Comment:
Looks like this hunk doesn't belong here.

Reply:
ahh, exactly. moved it to the place where it belongs to.

Reply:
I still see a change here, even though `locator/util.cc` the method is still called `describe_ring()`.

Reply:
i am sorry, should be good now.